lead-time/controller/lead_time_controller.py

```diff
@@ -1,10 +1,134 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, Header
+from typing import Annotated
+
+from pydantic import BaseModel
+from datetime import date
+from library.project.getProjectBySlug import get_project_by_slug
+from library.userStory.get_user_story_history import get_closed_user_stories
 
 router = APIRouter()
 
 
+class TimeRequest(BaseModel):
```

Comment:
Keep them as the same folder structure. Move this to models

Reply:
Modified